src/System.Runtime.InteropServices.RuntimeInformation/ref/System.Runtime.InteropServices.RuntimeInformation.cs

```diff
@@ -21,8 +21,20 @@ public partial struct OSPlatform : System.IEquatable<System.Runtime.InteropServi
         public static bool operator !=(System.Runtime.InteropServices.OSPlatform left, System.Runtime.InteropServices.OSPlatform right) { return default(bool); }
         public override string ToString() { return default(string); }
     }
+
+    public enum Architecture
+    {
+        x86,
+        x64,
+        ARM
+    }
+
     public static partial class RuntimeInformation
     {
+        public static Architecture GetProcessArchitecture() { return default(Architecture); }
+        public static Architecture GetOSArchitecture() { return default(Architecture); }
+        public static string GetOSDebugName() { return default(string); }
```

Comment:
In the APIReview I mentioned that I think it should be called GetOSDebugInfo because it isn't exactly a name and it isn't something that we want to encourage people to display (or parse) it is more for logging and debugging purposes. I still prefer DebugInfo over DisplayName because I don't believe DisplayName accurately represents what this contains and also doesn't help with people understanding of what it does. Without reading the documentation if I saw an API called GetOSDisplayName I would assume it would just give me something like "Windows" or "Linux" as in a short specific name but that isn't what this API will give you.

As for a property verse a method I'm fine with it being a read-only property, in fact that should probably be the case for all these methods.